testdata/project-v3/config/default/manager_config_patch.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: apps/v1
```

Comment:
Shows that we are missing for the addon. Should we not to do the same for the addon projects?

Reply:
Solved